src/riak_kv_qry_worker.erl

```diff
@@ -187,10 +187,8 @@ make_key_conversion_fun(Table) ->
     Mod = riak_ql_ddl:make_module_name(Table),
     DDL = Mod:get_ddl(),
     fun(Key) when is_binary(Key) ->
-            {ok, DescAdjustedKey} =
-                riak_ql_ddl:negate_if_desc(
-                  tuple_to_list(sext:decode(Key)), Mod, DDL),
-            {ok, PK} = riak_ql_ddl:lk_to_pk(DescAdjustedKey, Mod, DDL),
+            {ok, PK} = riak_ql_ddl:lk_to_pk(
```

Comment:
In contrast to its use in `riak_kv_ts_api:{get,delete}_data` where the key comes from user input, the folding functions here will have the key that's guaranteed to be valid (it is read directly from the backend).